### ISSUE TITLE
Not valid verfiication Code - Duplicate #10

### ISSUE THREAD

**Issue by ManWuzi** [Alan4747/instagram-auto-create-account#20]
@Alan4747  I experienced this when i tested it today. I think Instagram sends a faux code to fail first so you request a new one. My suggestion is to detect the error and the request another code, get the new code and complete it. 

Let me know if you'll work on it or will need my help (doubt you will, you've done a great job already)

Thank you for the project!

**Comment by ManWuzi** [comment]
Duplicate - #10

**Comment by Alan4747** [comment]
Yes, I guess it happens like you said, doubling happens. I'll be happy if you can help :)

**Comment by ManWuzi** [comment]
I'll try to work on it and make a pull request

**Comment by mikegrep** [comment]
I make pull request.If the owner accept it will be available soon.
